life_expectancy/tests/test_data_cleaning.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for the cleaning module"""
+import unittest.mock
+from unittest.mock import patch, Mock
+import pandas as pd
+import pytest
+from pytest import MonkeyPatch
+from life_expectancy.cleaning import main, load_data, clean_data, save_data
+from . import OUTPUT_DIR, FIXTURES_DIR
+
+
+def test_end_to_end(pt_life_expectancy_expected):
+    """Run the `clean_data` function and compare the output to the expected output"""
+    main("PT")
+    pt_life_expectancy_actual = pd.read_csv(
+        OUTPUT_DIR / "pt_life_expectancy.csv"
+    )
+    pd.testing.assert_frame_equal(
+        pt_life_expectancy_actual, pt_life_expectancy_expected
+    )
+
+
+@pytest.fixture(scope="session")
```

Comment:
Pylint is giving you a warning because of the way pytest fixtures work.
A good practice is to move the fixtures into conftest.py so they are not in the same module as the test code using the fixtures. Pytest load all fixtures on the `conftest.py` module by default, so this location is also useful for sharing fixtures among test modules.